priv/www/js/formatters.js

```diff
@@ -567,6 +567,16 @@ function fmt_shortened_uri(uri) {
     }
 }
 
+function fmt_uri_with_credentials(uri) {
+    if (typeof uri == 'string') {
+        // mask password
+        var mask = /^([a-zA-Z0-9\+\-\.]+):\/\/(.*):(.*)@/;
```

Comment:
Sorry to be pedantic, but `\` isn't necessary to escape special chars when they appear in a character class (`[ ]`):

![regex-without-backslashes](https://user-images.githubusercontent.com/514926/29847303-d2b4160e-8ccf-11e7-880a-2053d42903e8.png)